src/database/mongodb.ts

```diff
@@ -1,6 +1,6 @@
 import mongoose from 'mongoose';
 
-const ENDPOINT = process.env.ENDPOINT_DB || 'mongodb://mongodb:27017/club-manager';
+const ENDPOINT = process.env.ENDPOINT_DB || 'mongodb://127.0.0.1:27017/club-manager';
```

Comment:
Are you running on the docker container or a local instance? This breaks on my docker-compose